### DISCUSSION TITLE
Groups ,subgroups and subsubGroups

### DISCUSSION THREAD

**Discussion by FauconnierS** [Eugeny/tabby#6623] [Q&A]
Hello Dear,

I working with multiple environments with same domain and each environnment contains multiple server so multiple ssh session.
Is it possible to create subgroup to manage multiple session like "domain < environnment < ssh session".

![subgroups](https://user-images.githubusercontent.com/60675801/174111188-246bc37d-98d7-4dc2-8cdc-0bb1eaf4ff3b.png)

**Comment by ouyangxudu** [comment]
looking forward to...
